src/main/java/network/kngsgaming/minecraft/EnchantEvents.java

```diff
@@ -221,16 +220,17 @@ public void onInventoryClick(InventoryClickEvent event){
     }
 
     //Check to see if this enchantment and level are within the config defined limits
+    //Unused?
     private boolean isValidEnchantLevel(Enchantment enchantment, int level) {
         //assume no limit!
         int limit = Integer.MAX_VALUE;
         String enchanmentName = enchantment.getKey().getKey();
         if (limits.containsKey(enchanmentName)) {
-            limit  = 0;
             try {
-                //try to cast the object to an Integer
-                limit = (Integer) limits.get(enchanmentName);
-            } catch(Exception e) {
+                //parsing to an Integer
+                limit = Integer.parseInt(limits.get(enchanmentName));
```

Comment:
I actually considered using parseInt. The problem with how you have it here is that limits.get(enchantName) returns an object, so four that to work it would need to be Integer.parseInt(limits.get(enchanmentName.toString()));

Since I already know that from the config that Object is really an Integer, I think that the cast is more efficient than calling toString() and then parseInt(). I could be wrong, but I'm going to leave it as it is.

Reply:
Yes, you're right. It would be better. I also considered that